security/crowdsec/files/pkg-deinstall.in

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+case $2 in
+       DEINSTALL)
+               %%PREFIX%%/etc/rc.d/crowdsec stop
+               ;;
+       POST-DEINSTALL)
```

Comment:
The `POST-DEINSTALL` case could be used to clean extraneous leftovers such as crowdsec agent pid file, data dir, etc.. Basically everything created but not handled by the package files listing.

If it's not used at the moment, I would say not need to add it.